content/post/preFlutterhacks.md

```diff
@@ -0,0 +1,61 @@
+
+![Flutter-meme](https://memecreator.org/static/images/memes/4772258.jpg#center)
+
+## What are your learning approaches?
+
+So, developers try several different things in order to get started with any technology. Few of them quick start from the official documentation and then building apps directly. Then they look for support from community forums or websites like Stack Overflow.  The problem with this approach is that it is time-consuming. The developers have to spend more time in R&D instead of coding properly. Which sometimes came out as very fruitful and it is now more of a conventional but more of an easily available way of approaching.
+
+The other approach is developers plan out what they need to learn and they start the process by building a very basic app. The difference these developers have from the earlier one is these guys have access to quality content. More of a type of courses free/paid but they can start off their journey by the very first day of development. It is a proven fact that VUI and graphical ways of learning are more effective than conventional ways.
+
+Both of the approaches are good. It depends on you, How you wanna learn and What’s the more effective way you feel. Here in this blog, I will be talking about what are the skill set you require as a newcomer.
+
+## What else needs preparation?
+
+I have observed myself that I had to look for the same topics again and again while developing any app. As the development in Flutter is divided into two parts, **Screens and Widgets**. Most of the time building any application your development will run around these two major areas. But there are several other concepts and skills a developer should be aware of are:
+
+- Basics of rendering a screen with the help of MaterialApp, Scaffold and AppBar.
+- What are Forms and Lists in Flutter?
+- How the Flutter SDK compiles the same code for both iOS and Android platforms.
+- Basics of Android(Not necessarily but it will help) and experience with Object-oriented programing will help for sure.
+
+## How I started ??
+
+I am working with Flutter around for a year. I have started with the very same word that is most important for Flutter developers i.e. Widgets. I started using few basics widgets untill they boosted my confidence up and put me in a situation where I can appreciate my development skills on my own.
+
+There is a pool of widgets supported by Flutter today but **the key is to focus on the basic ones first**. I am suggesting a starter list which worked for me and I am sure it will work for you as a beginner:
+
+- MaterialApp, Scaffold and AppBar
+- Layout using Row & Column.
+- Text and Image widgets.
+- Containers and Padding, specifically the alignment.
+
+
+Yes, this is the most basic list a newcomer should cover before diving into deeper topics. Trust me you don't even realise how this basic approach will make you achieve.
```

Comment:
Refactor this content into the topics above and change the questions you are asking to accommodate those changes. You go something like this. **Optional**
```
**Why to X?**

You begin to explain X, 
The importance of X in Flutter, 
How you learned or did X and from where did you do it from.
```